src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -538,8 +544,17 @@ class Namespace:
     @staticmethod
     def init_namespace_dbs():
         db_conn = []
-        SonicDBConfig.load_sonic_global_db_config()
-        for namespace in SonicDBConfig.get_ns_list():
+        if not SonicDBConfig.isInit():
+            if multi_asic.is_multi_asic():
+                SonicDBConfig.load_sonic_global_db_config()
+            else:
+                SonicDBConfig.initialize()
+        # Ensure that db connector of default namespace is the first element of
+        # db_conn list.
+        db_conn.append(SonicV2Connector(use_unix_socket_path=True, namespace=multi_asic.DEFAULT_NAMESPACE, decode_responses=True))
```

Comment:
>namespace=multi_asic.DEFAULT_NAMESPACE, decode_responses=True [](http://example.com/codeflow?start=67&length=61)

We are using default values of `namespace` and `decode_responses`, so we can remove them.

Reply:
Updated as per comment.

Reply:
The unit test which uses sonic-py-swsssdk, has default namespace as None, but sonic-swsscommon has default namespace as empty string. So modified to specify default namespace.

Reply:
Modified to use namespace list from SonicDBConfig.get_ns_list().
get_ns_list will return list of namespace including default namespace as empty string.